src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleaseHelpAndSupportSection.tsx

```diff
@@ -129,6 +129,7 @@ const AccordionComponent = ({
 }: AccordionComponentProps) => {
   return includeAnalytics ? (
     <Accordion
+      visuallyHiddenText={`help and support sections for ${publicationTitle}`}
```

Comment:
As this is in the context of a publication page I don't think it needs to have the publication title in it, it could just be 'help and support sections'